src/coreclr/src/tools/Common/Internal/Runtime/ReadyToRunConstants.cs

```diff
@@ -314,6 +314,17 @@ public enum ReadyToRunHelper
         TypeHandleToRuntimeTypeHandle,
     }
 
+    // Enum used for HFA type recognition.
+    // Supported across architectures, so that it can be used in altjits and cross-compilation.
+    public enum ReadyToRunHFAElemType
+    {
+        None,
```

Comment:
Nit: I would use `None = 0` for consistency with the following lines and the `ReadyToRunHFAElemType` enumeration.